src/main/java/dev/dfonline/codeclient/config/Config.java

```diff
@@ -555,15 +555,15 @@ public YetAnotherConfigLib getLibConfig() {
                                 )
                                 .controller(integerOption -> IntegerFieldControllerBuilder.create(integerOption).range(0, 100))
                                 .build())
-                        .option(Option.createBuilder(Config.DestroyItemReset.class)
+                        .option(Option.createBuilder(DestroyItemReset.class)
```

Comment:
If you're looking to fix this, make sure the name of the option's variable on the top matches the name in that big list that serializes the options

Reply:
ah i see now, i was looking and looking and never noticed those names were different